src/concepts/components/styling.md

```diff
@@ -0,0 +1,36 @@
+# Styling
+
+&lt;Work in progress&gt;
+
+Proposal for proper CSS support can be found here: [https://github.com/yewstack/yew/issues/533](https://github.com/yewstack/yew/issues/533)
+
+
+In the meantime you can define the style with any standard CSS tools
+
+## Yew component does not support yet class attribute
+
+So far Yew component does not support natively the `class` attribute. You can add the class attribute to your component:
+
+```rust
+#[derive(Properties, Clone, PartialEq)]
+...
+pub struct Props{
+...
+#[prop_or_default]
+    pub class:String,
```

Comment:
I think there should be a space here. `pub class: String`

Reply:
ok, but not need anymore